src/components/Person/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const personsSpouse = sex === 'm' ? 'wife' : 'husband';
+  const partnerInfo = isMarried
+    ? `My ${personsSpouse}'s name is ${partnerName}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      <p className="Person__age">
+        {`I am ${age || 'unknown age'}`}
+      </p>
```

Comment:
Use conditional rendering
```suggestion
      {age && (
	      <p className="Person__age">
	        {`I am ${age}`}
	      </p>
      )}
```